styles/style.css

```
@@ -3,8 +3,116 @@ Colors to use:
 dark blue: #06365f
 light blue: #1c5380
 */
+@import url('https://fonts..com/css?family=Helvetica+Neue:300|Helvetica:100,300,700');
+
+* {
+  padding: 0;
+  margin: 0;
+}
 
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  background: url(/images/bgtile.png);
+  background-repeat: repeat-x;
```

```suggestion
  font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
  margin: 0;
  background-image: url(../images/bgtile.png);
  background-repeat: repeat-x;
```
Solo te faltaba ajustar la propiedad de `background-image`